### DISCUSSION TITLE
laravel-mail-editor not supporting in laravel 11

### DISCUSSION THREAD

**Discussion by in-vishal-chauhan** [community/community#130224] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello I am Vishal Chauhan
I have update laravel project to laravel version 11 and now 'laravel-mail-editor' library/module is not working.
Can you suggest me how can i use 'laravel-mail-editor' module laravel 11?

**Comment by ebndev** [comment]
Hi @in-vishal-chauhan, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Reply by in-vishal-chauhan** [reply]
thanks 
I only find this link to write my question from that github repo so i wrote here
Let me check if link is active on that github repo for discussion
